src/doblib/__main__.py

```diff
@@ -136,6 +150,12 @@ def main(args=None):
     elif args.command in ("a", "action"):
         # Run actions in the environment
         sys.exit(ActionEnvironment(args.cfg).apply_action(left))
+    elif args.command in ("m", "migrate"):
+        # Run Odoo migration using OpenUpgrade
+        migrate_args, left = load_migrate_arguments(left)
+        version = migrate_args.version
+        migrate_cfg = f"odoo.migrate.{version}.yaml"
```

Comment:
This differs from the scheme of the migrate files. You expect that the user enters e.g. `15.0` but the migrate files only use 15 but here you are using 15.0 in the file name.

Best would here to also use only the major version without minor.